components/empty-view/src/types.ts

```diff
@@ -1,17 +1,40 @@
-import { PropsWithChildren, ReactNode } from "react";
+import { HTMLAttributes, PropsWithChildren, ReactNode } from "react";
 
-import { Illustration } from "./constants";
-
-export type IllustrationKind = keyof typeof Illustration;
+export type IllustrationKind =
+  | "add-user-profile"
+  | "data"
+  | "devices"
+  | "empty-folder"
+  | "empty-space"
+  | "file-missing"
+  | "general"
+  | "no-access"
+  | "no-connection"
+  | "no-content"
+  | "no-match"
+  | "no-sites"
+  | "not-found"
+  | "settings"
+  | "success"
+  | "media"
+  | "team"
+  | "under-construction";
 
 export interface ContentProps {
   readonly body: ReactNode;
   readonly illustration: IllustrationKind;
   readonly title: string;
 }
 
-export type EmptyViewProps = PropsWithChildren<{
-  readonly after?: ReactNode;
-  readonly illustration: IllustrationKind;
-  readonly title: string;
-}>;
+export type EmptyViewProps = PropsWithChildren<
+  {
+    readonly after?: ReactNode;
+    readonly illustration: IllustrationKind;
+    readonly title: string;
+  } & HtmlDivAttributesRestProps
+>;
+
+export type HtmlDivAttributesRestProps = Pick<
+  HTMLAttributes<HTMLDivElement>,
+  "className" | "style"
```

Comment:
Do we need `style`?

Reply:
yes, i think its good too have. u never know :D